src/app/modules/shared/feature-toggles/feature-toggles.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Inject, Injectable, InjectionToken } from '@angular/core';
+import { AppConfigurationClient } from '@azure/app-configuration';
+import { from, Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { AZURE_APP_CONFIGURATION_CONNECTION_STRING } from 'src/environments/environment';
+import { FeatureToggle } from './toggle.model';
+
+
+const APP_CONFIGURATION_CLIENT = new InjectionToken<AppConfigurationClient>('Azure configuration client', {
+  providedIn: 'root',
+  factory: () => new AppConfigurationClient(AZURE_APP_CONFIGURATION_CONNECTION_STRING)
```

Comment:
Not sure if I should change the way I test this component so that the coverage report does not complain about not reaching this line on unit tests.

Reply:
It's OK, no worries